src/has.ts

```diff
@@ -76,20 +132,24 @@ add('weakmap', function () {
 		const map = new global.WeakMap([ [ key1, 1 ] ]);
 		return map.get(key1) === 1 && map.set(key2, 2) === map;
 	}
+	/* istanbul ignore next: node environment will never get here */
```

Comment:
Why are we doing this?  When the coverage is run in a combined CI way, we will cover this.